src/data/veilederbrukerknytning/useGetVeilederBrukerKnytning.tsx

```diff
@@ -8,6 +8,7 @@ export const veilederBrukerKnytningQueryKeys = {
     "veilederBrukerKnytning",
     personident,
   ],
+  historikk: (personident: string) => ["historikk", personident],
```

Comment:
Denne bør spesifiseres nærmere så vi ikke får potensiell krasj med queries definert andre steder:
```suggestion
  historikk: (personident: string) => ["historikk", "veilederBrukerKnytning", personident],
```